content/applications/hr/recruitment/recruitment_analysis.rst

```diff
@@ -0,0 +1,103 @@
+====================
+Recruitment analysis
+====================
+
+The *Recruitment Analysis* report allows recruiting departments to see which job positions have
+the most applicants, which have the most referrals, and how long it takes for applicants to move
+through the pipeline.
+
+Knowing how many applicants each specific job position has, along with statistics about how many are
+hired and refused, can provide valuable insights. This information can assist the recruiting team to
+pivot their strategies to acquire the most desirable candidates.
+
+Recruitment analysis report
+===========================
+
+Start by navigating to :menuselection:`Recruitment app --> Reporting --> Recruitment Analysis`.
+This presents a line chart of all applicants for the last year.
+
+Three separate color-coded metrics are presented: :guilabel:`In Progress`, :guilabel:`Hired`, and
+:guilabel:`Refused`.
```

Comment:
Do you think it would be beneficial or "not worth it" to mention that the 'Refused' line might not readily appear if there haven't been any refused applicants? Just something to consider during your revisions -- not a requirement.